testing_and_setup/compass/ocean/cosine_bell/RMSE/default/config_driver.xml

```diff
@@ -0,0 +1,5 @@
+<driver_script name="run.py">
+  <case name="rmse">
```

Comment:
Eventually, we'd want to add some `prerequisite` tags here.  It would be harmless to do so now if you want to.  See https://github.com/MPAS-Dev/MPAS-Model/pull/560